dockerfiles/php.dockerfile

```diff
@@ -15,20 +15,50 @@ COPY --from=composer:latest /usr/bin/composer /usr/local/bin/composer
 # MacOS staff group's gid is 20, so is the dialout group in alpine linux. We're not using it, let's just remove it.
 RUN delgroup dialout
 
+# Add the laravel user and group
 RUN addgroup -g ${GID} --system laravel
 RUN adduser -G laravel --system -D -s /bin/sh -u ${UID} laravel
 
+# Update php-fpm configuration to use the laravel user
 RUN sed -i "s/user = www-data/user = laravel/g" /usr/local/etc/php-fpm.d/www.conf
 RUN sed -i "s/group = www-data/group = laravel/g" /usr/local/etc/php-fpm.d/www.conf
 RUN echo "php_admin_flag[log_errors] = on" >> /usr/local/etc/php-fpm.d/www.conf
 
-RUN docker-php-ext-install pdo pdo_mysql
+# Install necessary packages
+RUN apk add --no-cache \
+    icu-dev \
+    zip \
+    libzip-dev \
+    libintl \
+    oniguruma-dev \
+    curl \
+    gcc \
+    make \
+    autoconf \
+    g++ \
+    libc-dev \
+    linux-headers \
+    && docker-php-ext-install \
+    pdo_mysql \
+    intl \
+    zip \
+    pcntl
 
+# Install Redis extension
+RUN pecl install redis && docker-php-ext-enable redis
```

Comment:
It could be:

RUN pecl install igbinary && \
    docker-php-ext-enable igbinary  && \
    echo "y" | pecl install redis && \
    docker-php-ext-enable redis